tmt/export/nitrate.py

```diff
@@ -84,22 +84,17 @@ def _nitrate_find_fmf_testcases(test: 'tmt.Test') -> Iterator[Any]:
     """
     import tmt.base
     assert nitrate
-    for component in test.component:
-        try:
+    with suppress(tmt.utils.StructuredFieldError, nitrate.NitrateError):
```

Comment:
Does this change make sense @happz? I'm thinking  
```
try:
    with suppress(tmt.utils.StructuredFieldError):

except nitrate.NitrateError:
    pass
``` 
but not sure if there is an advantage in doing so.

Reply:
changed in https://github.com/teemtee/tmt/pull/3403/commits/64f624261f0d734330586a9420c0055ec1b99a16